packages/robot/odahuflow/robot/libraries/model.py

```diff
@@ -44,21 +44,21 @@ def get_model_info(edge, token, model_name, model_version=None):
         :param token: model API JWT token
         :return: dict -- response
         """
-        headers = {"Authorization": "Bearer {}".format(token)}
+        headers = {"Authorization": f"Bearer {token}".format()}
```

Comment:
I suppose that there is no need for it)

```suggestion
        headers = {"Authorization": f"Bearer {token}"}
```